backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/MigrateVmCommand.java

```diff
@@ -613,6 +622,8 @@ public void runningSucceded() {
             updateVmAfterMigrationToDifferentCluster();
             plugPassthroughNics();
             initParametersForExternalNetworks(destinationVds, true);
+            setDedicatedCpus(getDestinationVdsManager());
+            vmDynamicDao.update(getVm().getDynamicData());
```

Comment:
the reason for calling vmDynamicDao.clearMigratingToVds(getVmId()) above rather than storing VmDynamic was to avoid a race with the monitoring - can we just change clearMigratingToVds to also store the dedicated cpus if specified?